Documentation/install/upgrade.rst

```diff
@@ -305,6 +305,11 @@ New Default Values
  * The connection-tracking garbage collector intervals is now 12 hours when
    using LRU maps (newer kernels) and 15 minutes an all older kernels. The
    interval can be overwritten with the option ``--conntrack-gc-interval``.
+   If connectivity between pods is faulty and ``cilium monitor --type drop``
+   shows ``xx drop (CT: Map insertion failed)`` it's advised to set up the
+   ``--conntrack-gc-interval`` to an interval lower than the default or the
```

Comment:
it's advised to set up --> it is recommended to set 

lower than the default or the -->lower than the default. Alternatively, the value for ``bpf-ct-global-any-max`` can be increased.